app/frontend/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,33 @@
+.navbar {
+  justify-content: center;
+  display: flex;
+  align-items: center;
+
+  .logo {
+    margin-right: 120px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 60px ;
+    gap: 30px;
+  }
+
+
+  nav li a {
+    margin-left: 20px;
+    color: #fff;
+    position: relative;
+    font-size: 20px;
+    display: flex;
+    align-items: center;
+    }
```

Comment:
Why are there 2 lines between these 2 selectors ?
And the indentation of the closing bracket of your `nav li a` selector isn't correct